app/models/outgoing_message_prototype.rb

```diff
@@ -160,17 +160,15 @@ def raw_message
       mail.sender = @sender
       mail.subject = @subject
       mail.reply_to = @reply_to
-      if @html_body.blank? && attachments.empty?
-        mail.body = @plain_body
```

Comment:
It seems like this outer condition that checks for plain text only messages should still exist. There's no reason to use `multipart/alternative` for those, right?

Reply:
I was initially erring on the side of caution but also the comment I linked to from the issue suggests that if you include a text/plain attachment without this arrangement, the attachment can get lost on some platforms.

Either way, its better to clearly define the parts within a message as theres no knowing what email software and servers will be involved in the passage of the message to its recipient.

Reply:
Works like a charm !